src/DucksNet.API/DTO/OfficeDTO.cs

```diff
@@ -3,6 +3,6 @@ namespace DucksNet.API.DTO;
 public class OfficeDTO
 {
     public Guid BusinessId { get; set; }
```

Comment:
this can not be empty?

Reply:
This dto isn't mine. I just changed the address to `set;` because I culdn't test one of my classes without it.

Reply:
This DTO isn't mine. I just changed to he Address to be `set;` because I couldn't test one of my classes without it.